app/components/user-status.hbs

```diff
@@ -5,9 +5,24 @@
 {{else}}
 <div class="heading">
   {{#each this.currentUserStatus as |currentStatus|}}
-    {{#if (eq @status currentStatus.status)}}
-      <h2 data-test-status>{{currentStatus.message}}</h2>
+  {{#if (eq @status currentStatus.status)}}
+    {{#if (eq currentStatus.status "ONBOARDING")}}
+      <div data-test-onboarding-details class="onboarding-details">
+        <h2 data-test-status>You are undergoing onboarding</h2>
+        <p data-test-details>
+          If you are a Developer, please complete your submission for 
+          <a href="https://github.com/Real-Dev-Squad/lift-simulation">Lift Simulation</a>, 
+          and ask for doubts in the team chat on Discord.
+        </p>
+        <p data-test-details>If you are not a developer, please contact Ankush for next steps.</p>
+        <button class="buttons__active" type="button"  {disabled={{@isStatusUpdating}} {{on "click" (fn
+          this.changeStatus "ACTIVE" )}} data-test-active-button>
+          <span>Change your status to Active</span>
+        </button>
+      </div>
     {{/if}}
+    <h2 data-test-status>{{currentStatus.message}}</h2>

```

Comment:
should this be in the `else`. i.e if the status is `onboarding` then we shouldn't show this. Right ?